### ISSUE TITLE
Slack Notification broken. 

### ISSUE THREAD

**Issue by Paul2e** [appveyor/ci#625]
Hi, 

The slack notification is currently failing with "channel_not_found" when attempting to post to Slack.

I have checked the status of Slack Web API, and it is correctly forwarding on messages to the correct channel.

We noticed that our notification stopped working on the 1st Feb 2016. We have not made any changes to our appveyor.yml. 

Thanks

Paul

**Comment by FeodorFitsner** [comment]
Hi Paul,
Add "#" to channel name to denote public channel.

-Feodor




On Mon, Feb 1, 2016 at 5:41 AM -0800, "Paul Touhey" <notifications@github.com> wrote:












Hi, 



The slack notification is currently failing with "channel_not_found" when attempting to post to Slack.



I have checked the status of Slack Web API, and it is correctly forwarding on messages to the correct channel.



We noticed that our notification stopped working on the 1st Feb 2016. We have not made any changes to our appveyor.yml. 



Thanks



Paul
